packages/backend/package.json

```diff
@@ -8,17 +8,21 @@
   "license": "GPL-3.0+",
   "scripts": {
     "dev": "webpack --watch --progress",
+    "generate-schema": "webpack --config schema.webpack.config.js --progress && node build/generateSchema.js",
```

Comment:
A script for running files with webpack would be good

https://github.com/sibelius/relay-workshop-private/blob/main/webpackx.ts